Antidote/AddFriendViewController.m

```diff
@@ -103,7 +147,18 @@ - (BOOL)textViewShouldBeginEditing:(UITextView *)textView
 {
     if ([textView isEqual:self.messageTextView]) {
         CGPoint offset = CGPointZero;
-        offset.y = 60.0;
+        if ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) {
+            offset.y = 60.0;
+        }
+        else {
+            if (SYSTEM_VERSION_LESS_THAN(@"8.0")) {
+                offset.y = 70.0;
+            }
+            else {
+                offset.y = 90.0;
```

Comment:
+ Fixed. Now offset.y calculate dynamically without information about orientation and system version.